templates/server.erb

```diff
@@ -0,0 +1,10 @@
+server <%= @name %> {
+  bogus <%= @bogus ? 'yes' : 'no' %>;
+  edns <%= @edns ? 'yes' : 'no' %>;
+<%- if @key -%>
+  keys <%= @key %>;
```

Comment:
According to this [page](http://www.zytrax.com/books/dns/ch7/server.html) current versions of bind don't support more than a single key.  I actually had it as an array until i tried it on a Scientific Linux 6.8 (bind 9.8.2) box and it didn't work.